core/domain/auth_services_test.py

```diff
@@ -231,3 +235,56 @@ def test_delete_association_when_it_is_present(self) -> None:
     def test_delete_association_when_it_is_missing_does_not_raise(self) -> None:
         # Should not raise.
         auth_services.delete_external_auth_associations('does_not_exist')
+
+    def test_auth_session_triggered(self) -> None:
+        def mock_establish_auth_session(
+                _: webapp2.Request,
+                __: webapp2.Response
+        ) -> None:
+            raise Exception('Is triggered')
+
+        def mock_destroy_auth_session(
+                _: webapp2.Response) -> None:
+            raise Exception('Is triggered')
+
+        with self.swap(
+                platform_auth_services,
+                'establish_auth_session',
+                mock_establish_auth_session
+        ):
+            with self.assertRaisesRegex(
+                    Exception, 'Is triggered'):
+                auth_services.establish_auth_session(
+                    webapp2.Request.blank('/'), webapp2.Response())
+        with self.swap(
+                platform_auth_services,
+                'destroy_auth_session',
+                mock_destroy_auth_session
+        ):
+            with self.assertRaisesRegex(
+                    Exception, 'Is triggered'):
+                auth_services.destroy_auth_session(webapp2.Response())
+
+    def test_super_admin_triggered(self) -> None:
```

Comment:
Try to make this name more descriptive/accurate, you aren't literally "triggering" the super admin (who is typically a user).

Reply:
Sure!